discord/ext/bridge/context.py

```diff
@@ -141,13 +141,9 @@ class BridgeExtContext(BridgeContext, Context):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self._original_response_message: Optional[Message] = None
```

Comment:
Why was this moved?

Reply:
Well this feature PR isn't just exclusively for `ext.bridge`, but it's also for `ext.commands`. So moving it would make sense.

Reply:
Moved it to Context so it could be used for the new `Context.delete()` method.